src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -143,25 +277,20 @@ def to_observe(self, event: Event) -> bool:
         this function is 2020-04-29 (because the time.step_size is 28
         days)
         """
-        return ((self.clock().year % 10 == 0)  # decennial year
-                and (self.clock().month == 4)  # month of April
-                and (self.clock().day <= self.time_step)  # time step containing first day of month
-               )
+        return (
+            (self.clock().year % 10 == 0)  # decennial year
```

Comment:
Note: we could clean up this logic (and support time-steps greater than 30 days with the following:
```
census_year = 10 * (self.event.year // 10)   # integer division is floor
census_date = datetime(census__year, 4, 1)
return self.clock() <= census_date < event.time
```

Reply:
Clever